### PR TITLE
Reclaim from sender channel

### PR DESCRIPTION
**Pull request by rhlsthrm**

### CODE REVIEW

File: modules/router/src/listener.ts [hunk 1]
```diff
@@ -166,12 +167,44 @@ export async function setupListeners(
...
+      );
+
+      const transferSenderResolutionChannelAddress = res.getValue()?.channelAddress;
+      const transferSenderResolutionTransferId = res.getValue()?.transferId;
+      if (!transferSenderResolutionChannelAddress || !transferSenderResolutionTransferId) {
+        logger.warn(
+          {
+            event: EngineEvents.CONDITIONAL_TRANSFER_RESOLVED,
+            transferSenderResolutionChannelAddress,
+            transferSenderResolutionTransferId,
```
Comment (LayneHaber) [inline modules/router/src/listener.ts:182]: should probably just log what the response is here

**Review by LayneHaber — APPROVED** [review]
some logging nits but otherwise lgtm

**Comment by LayneHaber** [comment]
Changelog

File: modules/router/src/listener.ts (hunk 1 above)
Comment (rhlsthrm) [reply modules/router/src/listener.ts:182]: We do two lines above.

Comment (LayneHaber) [reply modules/router/src/listener.ts]: ah should've expanded the preview, my bad

**Pull request merged**